Conflict with jnschulze webview_windows

When try to use with this plugin, we got the following error:

```shell
CMake Error at flutter/ephemeral/.plugin_symlinks/webview_windows/windows/CMakeLists.txt:33 (add_custom_target):
  add_custom_target cannot create target "DEPENDENCIES_DOWNLOAD" because
  another target with the same name already exists.  The existing target is a
  custom target created in source directory
  "C:/Users/dev/git/poc_issue_webview/windows/flutter/ephemeral/.plugin_symlinks/desktop_webview_auth/windows".
  See documentation for policy CMP0002 for more details.


CMake Error at flutter/ephemeral/.plugin_symlinks/webview_windows/windows/CMakeLists.txt:34 (add_custom_command):
  TARGET 'DEPENDENCIES_DOWNLOAD' was not created in this directory.
```
  
Steps to reproduce:

On new clean folder, open a new terminal and type in order:

`flutter create poc_issue_webview`
`cd poc_issue_webview`
`flutter pub add webview_windows`
`flutter pub add desktop_webview_auth`
`flutter run`

Select Windows as target 

Then you get the error.

https://github.com/jnschulze/flutter-webview-windows/issues/127

--- a/windows/CMakeLists.txt
+++ b/windows/CMakeLists.txt
@@ -29,9 +29,9 @@ if(NOT NUGET)
   endif()
 endif()
 
-add_custom_target(DEPENDENCIES_DOWNLOAD ALL)
+add_custom_target(${PROJECT_NAME}_DEPENDENCIES_DOWNLOAD ALL)
 add_custom_command(
-  TARGET DEPENDENCIES_DOWNLOAD PRE_BUILD
+  TARGET ${PROJECT_NAME}_DEPENDENCIES_DOWNLOAD PRE_BUILD
   COMMAND ${NUGET} install Microsoft.Windows.ImplementationLibrary -Version ${WIL_VERSION} -ExcludeVersion -OutputDirectory ${CMAKE_BINARY_DIR}/packages
   COMMAND ${NUGET} install Microsoft.Web.WebView2 -Version ${WEBVIEW_VERSION} -ExcludeVersion -OutputDirectory ${CMAKE_BINARY_DIR}/packages
   DEPENDS ${NUGET}

